net/onioncat/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                onioncat
+version             0.2.2.r569
+categories          net
+platforms           darwin
+maintainers         nomaintainer
```

Comment:
We have troubles with a huge number of non-maintained ports. Is there any strong reason why you don't want to be a maintainer of the port you are submitting (ideally with the `openmaintainer` mark)?